src/client/game/Player.js

```diff
@@ -67,20 +67,19 @@ Player.prototype.jump = function() {
 Player.prototype.move = function(direction) {
   if (direction === 'right') {
     this._direction = 'right'; 
-    this._sprite.body.velocity.x = 250;
+    this._sprite.body.velocity.x = this._WALK_SPEED;
   } else if (direction === 'left') {
     this._direction = 'left';
-    this._sprite.body.velocity.x = -250;
+    this._sprite.body.velocity.x = -1 * this._WALK_SPEED;
   }
 
   if (this._currentPlayingAnim === null || this._currentPlayingAnim.name.indexOf('walk') >= 0) {
-    this._currentPlayingAnim = this._sprite.animations.play('walk' + this._direction, this._WALK_SPEED);
+    this._currentPlayingAnim = this._sprite.animations.play('walk' + this._direction, this._WALK_FPS);
   }
 }
 
 /**
- * Sets the attack animation and shoots a projectile
- */
+ * Sets the attack animation and shoots a projeD/
```

Comment:
you didn't close this comment, so the `attack()` method has been commented out. Why did you change this line, anyway?